sys/include/net/ng_ipv6/nc.h

```diff
@@ -123,6 +124,23 @@ typedef struct {
     uint8_t l2_addr_len;                    /**< Length of ng_ipv6_nc_t::l2_addr */
     uint8_t flags;                          /**< Flags as defined above */
     kernel_pid_t iface;                     /**< PID to the interface where the neighbor is */
+    vtimer_t rtr_timeout;                   /**< timeout timer for router flag */
+
+    /**
+     * @brief (Re)Transmission timer for neigbor solicitations of this entry and
+     *        timeout for states.
+     */
+    vtimer_t nbr_sol_timer;
+
+    /**
+     * @brief Delay timer for neigbor advertisements of this entry.
```

Comment:
Because the line would be too long for inline-doc (the `/**<... */` thing). Semantically they are identical.